src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerMessageTest.cs

```diff
@@ -322,6 +323,33 @@ public void DefineScope_WithNoParameters_ThrowsException_WhenFormatString_HasNam
             Assert.Equal(expectedMessage, exception.Message);
         }
 
+        [Theory]
+        [MemberData(nameof(DefineMethodsData))]
+        public void DefineMessage_ThrowsException_WhenFormatString_IsNull(Delegate define)
```

Comment:
Realized that we can make `define` parameter into `Func<LogLevel, EventId, string, Delegate>` type instead of `Delegate`. Then we don't need `DynamicInvoke` call and check that the exception under the test has been wrapped with `TargetInvocationException`.

Reply:
Will push a commit with this change after we agree if we need to test the `ParamName` property of the exception